udata/core/dataset/rdf.py

```diff
@@ -468,6 +468,13 @@ def title_from_rdf(rdf, url):
             return i18n._("Nameless resource")
 
 
+def access_rights_from_rdf(resource: RdfResource):
+    top_el = resource.value(DCT.accessRights)
+    if top_el:
```

Comment:
I think `top_el` could be a `Literal`. Example: https://data.lillemetropole.fr/geonetwork/srv/fre/csw?SERVICE=CSW&VERSION=2.0.2&REQUEST=GetRecords&outputSchema=http://www.w3.org/ns/dcat%23&typenames=gmd:MD_Metadata&elementSetName=full&resultType=results

Reply:
I pretty much reused the license detection from the other PR, WDYT? https://github.com/opendatateam/udata/pull/3054/commits/c979244bef0be95d4f5668751d76de7f605f409a